app/lib/package/backend.dart

```diff
@@ -1201,7 +1201,7 @@ class PackageBackend {
   Future<void> _checkGithubActionAllowed(AuthenticatedGithubAction agent,
       Package package, String newVersion) async {
     final githubPublishing = package.automatedPublishing.github;
-    if (githubPublishing == null || (githubPublishing.isEnabled ?? false)) {
+    if (githubPublishing == null || !(githubPublishing.isEnabled ?? false)) {
```

Comment:
Maybe, this is easier to parse.
```suggestion
    if (githubPublishing?.isEnabled != true) {
```